server.lua

```diff
@@ -132,18 +138,20 @@ RegisterNetEvent('qb-storerobbery:server:openregister', function(IsDone)
 end)
 
 RegisterNetEvent('qb-storerobbery:server:trysafe', function()
-    local PlayerCoords = GetEntityCoords(GetPlayerPed(source))
+    local src = GetPlayerPed(source)
+    local PlayerCoords = GetEntityCoords(src)
     local ClosestSafeIndex = GetClosestSafe(PlayerCoords)
+    local Amount = QBCore.Functions.GetDutyCountType('leo')
```

Comment:
nit; camelCase local variable names
https://manason.github.io/effective-fivem-lua/variables/#camelcase-non-constant-local-variable-names

Reply:
nit; a more descriptive variable name could be good here. Maybe leoCount?

Reply:
Totally agree with you, will make these more descriptive, leoCount sounds good.